lib/autoyast.rb

```diff
@@ -267,20 +267,26 @@ def apply_changed_files(scope)
 
     @system_description[scope].files.each do |file|
       if file.deleted?
-        @chroot_scripts << "rm -rf '#{file.name}'"
+        @chroot_scripts << "rm -rf '#{Machinery.escape_single_quotes(file.name)}'"
```

Comment:
That escape call is quite long and used pretty often, but only with `file.name`. What about adding a method to `SystemFile` which returns the quoted name instead? (e.g. `file.quoted_name`)